checkbox-ng/checkbox_ng/launcher/stages.py

```diff
@@ -701,14 +701,16 @@ def _export_results(self):
                     result = result or {}
                     url = result.get("url") or result.get("status_url")
                     if url:
-                        path = self._get_submission_file_path(".c3_url.log")
-                        with open(path, "w+") as f:
+                        path = self._get_submission_file_path(
+                            ".submission.log"
+                        )
+                        with open(path, "a+") as f:
                             print(
-                                "Submission url ({}) saved also to: {}".format(
-                                    path, url
+                                "Submission url: {} (URL added to: {})".format(
+                                    url, path
```

Comment:
`Submission url:` adds a lot of verbosity for no practical gain, since it's gonna be printed for every exporter.

I've played a bit and came with this suggestion:

```suggestion
                                "[{}] {} (URL added to: {})".format(
                                    params["transport"], url, path
```

It leads to this kind of things:

```
[html_file] file:///home/pieq/.local/share/checkbox-ng/submission_2024-10-29T15.25.16.504567.html (URL added to: /home/pieq/.local/share/checkbox-ng/submission_2024-10-29T15.25.16.504567.submission.log)
[junit_file] file:///home/pieq/.local/share/checkbox-ng/submission_2024-10-29T15.25.16.504567.junit.xml (URL added to: /home/pieq/.local/share/checkbox-ng/submission_2024-10-29T15.25.16.504567.submission.log)
[tar_file] file:///home/pieq/.local/share/checkbox-ng/submission_2024-10-29T15.25.16.504567.tar.xz (URL added to: /home/pieq/.local/share/checkbox-ng/submission_2024-10-29T15.25.16.504567.submission.log)
Do you want to submit 'upload to certification-staging' report?
  y => yes
  n => no
[yn]: y
Enter submission description (press Enter to skip): ...
Enter secure-id:...
[c3] https://certification.staging.canonical.com/submissions/status/37897 (URL added to: /home/pieq/.local/share/checkbox-ng/submission_2024-10-29T15.25.16.504567.submission.log)
```

Reply:
good idea, done